.pre-commit-config.yaml

```diff
@@ -10,20 +10,13 @@ repos:
         args: [--fix=lf]
       - id: end-of-file-fixer
 
-  # - repo: https://github.com/asottile/pyupgrade
-  #   rev: v2.29.0
-  #   hooks:
-  #     - id: pyupgrade
-  #       args: [--py36-plus]
-  #       name: Upgrade code
-
```

Comment:
It seems that it was decided in #4585 to not use this hook, but it was kept here as a comment. Was this simply forgotten / overlooked and is safe to be removed?

Reply:
we can remove it